launcher/spec/launch_policy_test.go

```diff
@@ -0,0 +1,143 @@
+package spec
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestLauncPolicy(t *testing.T) {
+	testCases := []struct {
+		testName       string
+		imageLables    map[string]string
+		expectedPolicy LaunchPolicy
+	}{
+		{
+			"case 1",
```

Comment:
nit: perhaps instead of case 1 we have the test named single Env value.
for case 2, test multiple Env values.

Reply:
better names, done